lib/grape/path.rb

```diff
@@ -22,11 +22,11 @@ def root_prefix
     end
 
     def uses_specific_format?
-      !!(settings[:format] && Array(settings[:content_types]).size == 1)
+      (settings[:format] && Array(settings[:content_types]).size == 1)
     end
 
     def uses_path_versioning?
-      !!(settings[:version] && settings[:version_options][:using] == :path)
+      (settings[:version] && settings[:version_options][:using] == :path)
```

Comment:
I think keeping a double-negation is fine. I also don't agree with Rubocop here :)